src/login/login.component.tsx

```diff
@@ -84,23 +93,50 @@ export default function Login(props: LoginProps) {
                 value={username}
                 onChange={evt => setUsername(evt.target.value)}
                 placeholder="Username"
+                ref={usernameInputRef}
                 autoFocus
               />
             </label>
           </div>
-          <div>
+          <div
+            css={css`
+              position: relative;
+            `}
+          >
             <label>
               <input
                 css={css`
                   ${input}
                 `}
-                type="password"
+                type={showPassword ? "text" : "password"}
                 name="password"
                 value={password}
                 onChange={evt => setPassword(evt.target.value)}
                 placeholder="Password"
+                ref={passwordInputRef}
               />
             </label>
+            <button
```

Comment:
An example of an icon button with an "unstyled" button. Unstyled buttons don't have a hover state -- we'll likely for other button icons want to introduce a button class specifically that gives the icon hover state.